src/promptflow-evals/samples/built_in_evaluators.py

```diff
@@ -9,17 +9,14 @@
 from promptflow.entities import AzureOpenAIConnection
 
 model_config = AzureOpenAIConnection(
```

Comment:
rename it to `connection`?

Reply:
model config is more generic since for local run you can just pass model config and may not even need a connection. Maybe later on we can even support a generic dict with config info ?

Reply:
Since `AzureOpenAIConnection` doesn't have model/deployment property, calling `model_config` at least at this moment is a little confusing.